website/azurerm.erb

```diff
@@ -457,10 +457,6 @@
                 <li<%= sidebar_current("docs-azurerm-datasource-virtual-network-gateway-connection") %>>
                     <a href="/docs/providers/azurerm/d/virtual_network_gateway_connection.html">azurerm_virtual_network_gateway_connection</a>
                 </li>
-
```

Comment:
I'm not sure this will remove it from the new registry docs? 🤔

Reply:
I've raised this with the team separately / via Slack